style.css

```diff
@@ -1,2 +1,86 @@
+.credit-card  {
+    width: 500px;
+    height: 325px;
+    background-color: royalblue;
+    border-radius: 20px 20px 20px 20px;
+    display: flex;
+    flex-direction: column;
+    font-family: "Geologica", sans-serif;
+    font-weight: 300;
+    font-optical-sizing: auto;
+}
 
+.credit-card-front {
+    display: flex;
+    justify-content: flex-end;
+}
 
+.credit-card-front .information-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 0 0 10px 39px;
+    font-size: 20px;
+    width: 20%;
+    height: 20%
+}
+
+.information-wrapper > p:nth-child(1) {
+    font-weight: lighter;
+    text-transform: uppercase;
+}
+
+.information-wrapper > p:nth-child(2) {
+    margin-left: 10px;
+    font-size: 30px;
+}
```

Comment:
You could simplify `.information-wrapper > p:nth-child(x)` to just `.information-wrapper p:nth-child`.
Do you know why?

Reply:
Becasue the <p> elements are only children of <information-wrapper> ?